RELEASE/scripts/autoscend/combat/auto_combat_default_stage2.ash

```diff
@@ -3,6 +3,13 @@ string auto_combatDefaultStage2(int round, monster enemy, string text)
 	// stage 2 = enders: escape, replace, instakill, yellowray and other actions that instantly end combat
 	string retval;
 	
+	//if we want to olfact in stage 4 then we should delay using this for now until we have olfacted.
+	if(auto_wantToSniff(enemy, my_location()) && getSniffer(enemy) != $skill[none])
+	{
+		auto_log_debug("Skipping stage 2 of combat for now as we intend to olfact [" +enemy+ "]");
```

Comment:
General question - will we use multiple sniffers on a single enemy? If we do, is that what we want to do? Feel like I've asked this question before, but my memory is hazy.

Reply:
I don't think we should use more than 1 sniffer per target.
working on fixing this now. this is bigger than initially expected.